data/adapters/coxpresdb_adapter.py

```diff
@@ -0,0 +1,42 @@
+
+from adapters import Adapter
+import pickle
+import os
+
+# https://coxpresdb.jp/download/Hsa-r.c6-0/coex/Hsa-r.v22-05.G16651-S235187.combat_pca.subagging.z.d.zip
+# There is 16651 files. The file name is entrez gene id. The total genes annotated are 16651, one gene per file, each file contain logit score of other 16650 genes.
+# There are two fields in each row: entrez gene id and logit score
+
+
+class Coxpresdb(Adapter):
+
+    def __init__(self, file_path):
+
+        self.file_path = file_path
+        self.dataset = 'coxpresdb'
+        self.label = 'coxpresdb'
+        self.source = 'CoXPresdb'
+        self.source_url = 'https://coxpresdb.jp/'
+
+        super(Coxpresdb, self).__init__()
+
+    def process_file(self):
+        with open('./samples/entrez_ensembl.pkl', 'rb') as f:
```

Comment:
I think we should move this to a different folder since it's required for data loading and not a sample for dev testing.

Also just a quick comment on how this pkl was generated, or in the future if we have to modify we might not remember.